core/services/feeds/orm.go

```diff
@@ -553,6 +554,21 @@ WHERE id = $1;
 	return &spec, errors.Wrap(err, "CreateJobProposalSpec failed")
 }
 
+// GetApprovedSpec gets the approved spec for a job proposal
+func (o *orm) GetApprovedSpec(jpID int64, qopts ...pg.QOpt) (*JobProposalSpec, error) {
+	stmt := `
+SELECT id, definition, version, status, job_proposal_id, status_updated_at, created_at, updated_at
+FROM job_proposal_specs
+WHERE status = $1
+AND job_proposal_id = $2
+`
+
+	var spec JobProposalSpec
+	err := o.q.WithOpts(qopts...).Get(&spec, stmt, JobProposalStatusApproved, jpID)
```

Comment:
slight nit, I noticed there's two statuses sometimes used interchangeably but are also identical in string, `SpecStatus` and `JobProposalStatus`. Since this is on the specs table, if we are to keep both instead of just getting rid of one, then here it should probably be `SpecStatusApproved`

However, maybe it makes sense to merge them into one